test/public/defaults.js

```diff
@@ -439,3 +439,14 @@ module.exports.checkMismatchingUrlParam = async (page, expectedUrlParameters) =>
     }
     return ret;
 };
+
+/**
+ * Wait For table to reload data
+ * Reloaded table MUST contain at least one row of data
+ * @param {puppeteer.Page} page the puppeteer page
+ * @param {function} triggerFunction table reload trigger
+ * @return {Promise} promise
+ */
+module.exports.waitForTableDataReload = (page, triggerFunction) =>
+    Promise.all([page.waitForSelector('table .atom-spinner'), triggerFunction()])
+        .then(() => page.waitForSelector('tbody tr td:nth-child(2)'));
```

Comment:
I would rather wait for `.atom-spiner` to not be there anymore. This remove the limitation, because we have table in which we expect only one row (for example the `no data` message)

Reply:
added parameter controlling whether it waits for table with data

Reply:
This is not what I had in mind. This will not work when the table has only one row. 
I suggest to use `hidden: true` parameter for `waitForSelector`

Reply:
it will work because it looks for second cell of the first row not a second row, but yes, `hidden: true` for spinner is better